src/Microsoft.Identity.Web.TokenAcquisition/TokenAcquisitionExtensionOptions.cs

```diff
@@ -1,49 +1,23 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Identity.Abstractions;
 using Microsoft.Identity.Client;
 
 namespace Microsoft.Identity.Web
 {
-    /// <summary>
-    /// Signature for token acquisition extensions that act on the application builder.
-    /// </summary>
-    /// <param name="confidentialClientApplicationBuilder">Application builder.</param>
-    /// <param name="acquireTokenOptions">Token acquisition options for the request. Can be null.</param>
-    public delegate void BuildApplication(ConfidentialClientApplicationBuilder confidentialClientApplicationBuilder, AcquireTokenOptions? acquireTokenOptions);
-
-    /// <summary>
-    /// Signature for token acquisition extensions that act on the request builder, for an app token
-    /// </summary>
-    /// <param name="builder">Builder</param>
-    /// <param name="acquireTokenOptions">Token acquisition options for the request. Can be null.</param>
-    public delegate void BeforeTokenAcquisitionForApp(AcquireTokenForClientParameterBuilder builder, AcquireTokenOptions? acquireTokenOptions);
-
-    /// <summary>
-    /// Signature for token acquisition extensions that act on the application builder.
-    /// </summary>
-    /// <param name="authResult">MSAL.NET authentication result</param>
-    /// <param name="acquireTokenOptions">Token acquisition options for the request. Can be null.</param>
-    public delegate void AfterTokenAcquisition(AuthenticationResult authResult, AcquireTokenOptions? acquireTokenOptions);
-
     /// <summary>
     /// Options for TokenAcquisition add-ins. These options consist in a set of events, that can be subscribed to by add-ins
     /// or parts of the add-ins.
     /// </summary>
-    public class TokenAcquisitionAddInOptions
+    public class TokenAcquisitionExtensionOptions
     {
+#if FUTURE
```

Comment:
what does "FUTURE" mean? maybe we can have a better word here?

Reply:
CC @jmprieur